src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -8424,6 +8424,38 @@ def test_vmss_cross_tenant_update(self, resource_group, another_resource_group):
         ])
 
 
+class VMCreateFromACGToOtherSubScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_create_from_acg_image_to_other_sub')
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_create_from_acg_image_to_other_sub',
+                           parameter_name='another_resource_group', subscription='6b085460-5f21-477e-ba44-1035046e9101')
```

Comment:
Please do not create test resources under subscription `Azure SDK Infrastructure`, because this subscription is used to store online resources. 
Please refer to this subscription `Azure CLI Tests with TTL = 2 Days` used to test cross tenant/subscription scenarios from `from azure.cli.testsdk.constants import AUX_SUBSCRIPTION, AUX_TENANT`

Reply:
In addition, please mark the test as `@live_only`, because the CI pipeline currently does not support cross tenants